packages/number-field/package.json

```diff
@@ -75,7 +75,7 @@
     },
     "devDependencies": {
         "@formatjs/intl-numberformat": "^8.3.5",
-        "@spectrum-css/stepper": "7.0.0-s2-foundations.19"
+        "@spectrum-css/stepper": "7.1.3"
```

Comment:
Please check this version. This surfaced up a lot of `--mod` overrides which we need to revert back.